aws/aws.go

```diff
@@ -14,30 +14,9 @@ import (
 // colorizer
 var style aurora.Aurora
 
-func checkCredentialsEnvVar() bool {
-
-	if os.Getenv("AWS_ACCESS_KEY_ID") == "" || os.Getenv("AWS_SECRET_ACCESS_KEY") == "" {
-
-		return false
-
-	} else if os.Getenv("AWS_REGION") == "" {
-
-		if os.Getenv("AWS_DEFAULT_REGION") == "" {
-			return false
-		}
-		os.Setenv("AWS_REGION", os.Getenv("AWS_DEFAULT_REGION"))
-
-	}
-	return true
-}
-
 // ReadFile reads a file from S3 bucket and saves it in a desired location.
 func ReadFile(bucketName string, filename string, outFile string, noColors bool) {
 	style = aurora.NewAurora(!noColors)
-	// Checking env vars are set to configure AWS
-	if !checkCredentialsEnvVar() {
-		log.Fatal(style.Bold(style.Red("ERROR: Failed to find the AWS env vars needed to configure AWS. Please make sure they are set in the environment.")))
```

Comment:
thanks @jkrukoff-cb for the PR .. since the validation already sets AWS_REGION env var to support for users who have AWS_DEFAULT_REGION set instead, removing it can introduce a minor breaking change. 
I think it would be better to make the validation optional, i.e. replace the log.Fatal with log.Println saying that no environment variables are being used. Then the sdk can use any other alternatives that are available in the machine.